packages/jobs/lib/execution/sync.ts

```diff
@@ -99,8 +107,6 @@ export async function startSync(task: TaskSync, startScriptFn = startScript): Pr
             });
         }
 
-        await addSyncConfigToJob(syncJob.id, syncConfig.id!);
```

Comment:
Seems completely unnecessary to do it in two steps, probably a legacy thing

Reply:
indeed